client/package.json

```diff
@@ -29,11 +29,12 @@
     "classnames": "^2.2.6",
     "css-loader": "^3.4.2",
     "draft-js": "^0.11.5",
-    "fuse.js": "^5.1.0",
     "emotion-theming": "^10.0.27",
     "firebase": "^7.13.1",
     "font-awesome": "^4.7.0",
+    "fuse.js": "^5.1.0",
     "isomorphic-unfetch": "^3.0.0",
+    "jquery": "^3.4.1",
```

Comment:
remove this and react-tiny-popover

Reply:
ah yes the short-lived tale of tiny popover